src/pages/staff/dashboard/MyBrandsPage.jsx

```
@@ -60,8 +62,18 @@ export default function MyBrandsPage() {
       } catch (err) { console.debug?.('track page_view failed (effect path, my_brands_desktop)', err); }
     }
   }, [isDesktop]);
+
+  // Section view analytics when search toggles between empty/non-empty
+  useEffect(() => {
+    const section = (searchQuery || '').trim() ? 'search_results' : 'following';
+    if (lastSectionRef.current !== section) {
+      try { track('section_view', { page: 'my_brands', section }); } catch {}
+      lastSectionRef.current = section;
+    }
+  }, [searchQuery]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)